src/server/game/Entities/Player/Player.cpp

```diff
@@ -20812,7 +20821,7 @@ void Player::TextEmote(std::string_view text, WorldObject const* /*= nullptr*/,
 
     WorldPacket data;
     ChatHandler::BuildChatPacket(data, CHAT_MSG_EMOTE, LANG_UNIVERSAL, this, this, _text);
-    SendMessageToSetInRange(&data, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), true, !GetSession()->HasPermission(rbac::RBAC_PERM_TWO_SIDE_INTERACTION_CHAT));
+    SendChatMessageToSetInRange(&data, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_YELL), true, !GetSession()->HasPermission(rbac::RBAC_PERM_TWO_SIDE_INTERACTION_CHAT));
```

Comment:
what does this CONFIG_ change do ?

Reply:
My bad. I'll switch CONFIG_s in ::Yell and ::TextEmote

Reply:
Fixed in last commit